src/Navigation/styled.js

```diff
@@ -0,0 +1,33 @@
+import { NavLink } from "react-router-dom";
+import styled from "styled-components";
+
+export const Navigation = styled.nav`
+  width: 1920px;
```

Comment:
Nie było by lepiej jak by zamiast width: 1920px; dać max-width: 1920px; ??

Reply:
A ja bym proponował dać width 100% bo jeśli ktoś ma monitor szerszy niż 1920px to mu się nawigacja nie ma rozciagnie i będzie puste miejsce, nadziałem się na to już raz przy pracy domowej